mantatail.py

```diff
@@ -242,6 +242,15 @@ def set_operator(self, user: UserConnection) -> None:
     def remove_operator(self, user: UserConnection) -> None:
         self.operators.discard(user.nick.lower())
 
+    def is_founder(self, user: UserConnection) -> bool:
+        """
+        Takes a UserConnection object as an argument and checks if the user
+        is the channel founder.
+
+        Returns a Boolean.
+        """
```

Comment:
I don't like repeating what the type annotations already say. You should assume that the reader knows how to read type annotations, because your code is full of them anyway.

```suggestion
        """Checks if the user is the channel founder."""
```